test/tx-builder/merkle-distributor.spec.ts

```diff
@@ -0,0 +1,48 @@
+import { expect } from 'chai';
+import hre from 'hardhat';
+
+import {
+  Network,
+  TxBuilder,
+} from '../../src';
+import { NetworkName } from '../../src/types';
+import { describeContract, describeContractForNetwork, TestContext } from '../helpers/describe-contract';
+
+let txBuilder: TxBuilder;
+
+function init(ctx: TestContext): void {
+  txBuilder = new TxBuilder(
+    {
+      network: Network.hardhat,
+      hardhatMerkleDistributorAddresses: {
+        MERKLE_DISTRIBUTOR_ADDRESS: ctx.merkleDistributor.address,
+      },
+      injectedProvider: hre.ethers.provider,
+    },
+  );
+}
+
+describeContract('MerkleDistributor', init, (ctx: TestContext) => {
+  describeContractForNetwork(
+    'MerkleDistributor',
+    ctx,
+    NetworkName.hardhat,
+    true,
+    () => {
+      it('getActiveUsersInEpoch 0', async () => {
+        const activeUsers = await txBuilder.merkleDistributorService.getActiveUsersInEpoch(0);
+        expect(activeUsers.length).to.be.eq(32457);
+      });
+
+      it('getActiveUsersInEpoch 1', async () => {
+        const activeUsers = await txBuilder.merkleDistributorService.getActiveUsersInEpoch(1);
+        expect(activeUsers.length).to.be.eq(9979);
+      });
```

Comment:
Small nit: For validation reasons, can we add a test that `getActiveUsersInEpoch 5` returns what we expect?